File: java/sdk/event/input/user/ReleaseAccepted.java
package com.fleetmgr.client.sdk.event.input.user;

/**
 * Created by: Bartosz Nawrot
 * Date: 18.09.2018
 * Description:
 */
public class ReleaseAccepted extends UserEvent {

    private byte[] data;

    public ReleaseAccepted(byte[] data) {
        super(Type.RELEASE_ACCEPTED);
        this.data = data;
    }

    public byte[] getData() {
        return data;
    }
}

File: cpp/source/event/input/user/ReleaseAccepted.cpp
#include "event/input/user/ReleaseAccepted.hpp"

using namespace fm;
using namespace fm::event::input::user;

ReleaseAccepted::ReleaseAccepted() :
    UserEvent(RELEASE_ACCEPTED)
{
}

std::string ReleaseAccepted::toString() const
{
    return "RELEASE_ACCEPTED: ";
}
